a3p-integration/proposals/README.md

```diff
@@ -0,0 +1,96 @@
+# Draft proposals
+
+## How to publish a proposal once approved
+
+`a3p-integration` holds draft proposals to test in agoric-sdk that they perform
+as expected. Once a proposal has been approved by BLD stakers it executes on
+chain and the proposal is no longer a draft. That requires moving it to the
+agoric-3-proposals repo to become part of the `latest` image.
+
+Steps:
+1. Get the upgrade branch
+2. Build the submissions
+3. Migrate the proposal
+4. Update the `latest` iamge
+
+### Get the proposal's branch
+
+We need the actual proposal that was sent to stakers. In the case of upgrades
+(chain-halting) it will be a dedicated release branch. (E.g.
+https://github.com/Agoric/agoric-sdk/tree/dev-upgrade-16/). If you're
+publishing an "upgrade" proposal and it's called `a:upgrade-next`, keep looking;
+that is only a draft.
+
+### Build the submissions
+
+The proposals use `sdk-generate` to ensure that CI is always testing the latest
+version. But to publish we need a fixed version, to match the software upgrade
+that the BLD stakers decided on.
+
+To build that artifact,
+```
+cd a3p-integration
+scripts/build-all-submissions.sh
```

Comment:
This results in files with absolute filenames on the developer's machine; for example `/home/connolly/.agoric/cache/b1-...`. Should those be changed? Some advice one way or the other would be nice here.